Taeryoung/app.js

```diff
@@ -0,0 +1,30 @@
+const http = require ("http");
+const express = require ("express");
+const cors = require ("cors");
+const morgan = require ("morgan");
+const dotenv = require("dotenv");
+dotenv.config()
+
+app = express()
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan('combined'));
+
+
+// app.get ("/ping", (req, res) => {
+//     res.json({ message : "pong"});
+// });
+
+app.get('/ping', cors(), function (req, res, next) {
+    res.json({message: 'pong'})
+})
+
+const server = http.createServer(app)
+const PORT = process.env.PORT;
+
+const start = async () => {
+    server.listen(PORT, () => console.log(`server is listening on ${PORT}`));
+}
```

Comment:
`server`라는 변수에 할당된 서버를 띄울 수 있는 객체는 http 내장 모듈을 이용한 것이기 때문에 앞서 리뷰드린대로 해당 프로젝트에서는 불필요한 코드입니다.

이에 따라 8번 라인에서 선언한 `app` 이라는 변수를 이용하여 서버를 띄워야 합니다.
```suggestion
const start = async () => {
    app.listen(PORT, () => console.log(`server is listening on ${PORT}`));
}
```